### ISSUE TITLE
Remove publish.js in all examples

### ISSUE THREAD

**Issue by TimothyJones** [pact-foundation/pact-js#747]
Many examples have `publish.js`, which is sometimes a source of bugs or confusion. We should update the examples to use the pact-broker CLI, since it's more idomatic and more straightforward:

https://github.com/pact-foundation/pact-js#publish-in-npm-scripts

For example, in the jest example, [we have](https://github.com/pact-foundation/pact-js/blob/54de92fffd3a300deeb58a85a4ec8605c907c2b4/examples/jest/package.json)

```
"publish:pact": "node ./publish.js"
```

where publish.js is some 50 lines of javascript.

We can replace all 50 lines and remove publish.js by making the script run:

```
"pact:publish": "pact-broker publish pact/pacts --consumer-app-version=\"$(npx @pact-foundation/absolute-version)\" --tag-with-git-branch --broker-base-url=https://test.pact.dius.com.au"
```

and passing the username / password via `PACT_BROKER_PASSWORD` and `PACT_BROKER_USERNAME`

**Comment by aussiDavid** [comment]
To clarify, you want a file like [pact-js/examples/jest/package.json](https://github.com/pact-foundation/pact-js/blob/54de92fffd3a300deeb58a85a4ec8605c907c2b4/examples/jest/package.json) which has this script
`"publish:pact": "node ./publish.js"`
To look like 
`"pact:publish": "pact-broker publish <YOUR_PACT_FILES_OR_DIR> --consumer-app-version=\"$(npx @pact-foundation/absolute-version)\" --tag-with-git-branch --broker-base-url=https://your-broker-url.example.com"`
with all the recorrect values in the placeholders?

**Comment by TimothyJones** [comment]
That's right. I've edited the issue to make it clearer

**Comment by CalCorbin** [comment]
Created a PR for this issue at https://github.com/pact-foundation/pact-js/pull/753

**Comment by TimothyJones** [comment]
Closed by #753. Thanks so much, @CalCorbin!
